### PR TITLE
Update settings.yml

### PR DESCRIPTION
**Pull request by RedHatCRob**
suggested description for wg

### CODE REVIEW

**Review by xcorail — APPROVED** [review]

**Comment by xcorail** [comment]
Closes #25

**Comment by kaywilliams** [comment]
Thanks @xcorail and @RedHatCRob,

Let's include the spelled out working group name 'Best Practices for OSS Developers' in the description.  This is helpful for when the repository is pinned on the GH organization page.  Revised text "The Best Practices for OSS Developers working group is dedicated to raising awareness and education of secure code best practices for open source developers."

Does that work?

File: .github/settings.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ repository:
   name: wg-best-practices-os-developers
 
   # A short description of the repository that will show up on GitHub
-  description: OSSF Working group: secure code best practices for open source developers
+  description: Dedicated to raising awareness and education of secure code best practices for open source developers
```
Comment (xcorail) [inline .github/settings.yml:8]:
```suggestion
  description: The Best Practices for OSS Developers working group is dedicated to raising awareness and education of secure code best practices for open source developers.
```

**Comment by xcorail** [comment]
I updated the PR with your suggestion @kaywilliams

**Comment by kaywilliams** [comment]
Looks good. Thanks Xavier!

**Pull request merged**